website/docs/r/job.html.md

```diff
@@ -186,6 +188,14 @@ A command's `step_plugin` or `node_step_plugin` block both have the following st
 
 * `config`: (Optional) Map of arbitrary configuration parameters for the selected plugin.
 
+A command's `errorhandler` block both have the following structure:
+
+* `type`: (Required) The name of the plugin to execute.
+
+* `config`: (Optional) Map of arbitrary configuration parameters for the selected plugin.
+
+* `keepgoingonsuccess`: (Optional) A boolean to configure if the error handler job should be launched on the success of the job.
```

Comment:
I think some conventions are missing here.

`keepgoingonsuccess` should be `keep_going_on_success`